packages/jest-worker/package.json

```
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@types/node": "*",
+    "jest-util": "^29.1.0",
```

Would be better (and refresh the lock file, please):

```suggestion
    "jest-util": "workspace:^",
```